src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/jvmsizing/resources/rca.conf

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Do we need this full config file? Is it possible to just pass the test config as a string [here](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/RcaConf.java#L101-L109)?

Reply:
I just wanted to have full confidence in the framework being able to read the relevant sections of the config file and apply them correctly.